src/shared/parameter_v2/config/examples/parameter.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <stdbool.h>
+// Parameter for C
```

Comment:
super nit: add blank line between `#include` and start of comment

Reply:
added